src/shipit_code_coverage/shipit_code_coverage/report_generators.py

```diff
@@ -130,16 +131,19 @@ def zero_coverage(self, artifacts, out_dir='code-coverage-reports'):
 
         filesinfo = self.get_fileinfo(zero_coverage_functions.keys())
 
-        zero_coverage_report = []
+        zero_coverage_files = []
         for fname, functions in zero_coverage_functions.items():
             info = filesinfo[fname]
             info.update({'name': fname,
                          'funcs': len(functions),
                          'uncovered': fname in zero_coverage_files})
-            zero_coverage_report.append(info)
+            zero_coverage_files.append(info)
 
             with open(os.path.join(out_dir, 'zero_coverage_functions/%s.json' % fname.replace('/', '_')), 'w') as f:
                 json.dump(functions, f)
 
+        zero_coverage_report = {'revision': self.revision,
```

Comment:
If we store both the mercurial and the github revision, we won't need to query the mapper on the frontend.

Reply:
We already retrieve the github_revision at https://github.com/mozilla-releng/services/blob/master/src/shipit_code_coverage/shipit_code_coverage/codecov.py#L60, so you could just make it a property of the object so it can be accessed when calling report_generators.ZeroCov.